data/virt_autotest/guest_unattended_installation_files/opensuse_tumbleweed_xen_hvm_guest_x86_64.xml

```diff
@@ -174,5 +174,17 @@
       <username>root</username>
     </user>
   </users>
+  <scripts>
+    <!-- permit root login, password login and pubkeys login -->
+    <init-scripts config:type="list">
+      <script>
+        <source><![CDATA[
+echo -e "PermitRootLogin yes\nPubkeyAuthentication yes\nPasswordAuthentication yes\nAuthorizedKeysFile      .ssh/authorized_keys" > /etc/ssh/sshd_config
+[ -d /root/.ssh ] || mkdir /root/.ssh; echo "##Authorized-Keys##" >> /root/.ssh/authorized_keys
+]]>
```

Comment:
I think you'd better refer to this article:
https://developers.redhat.com/blog/2018/11/02/how-to-manually-copy-ssh-keys-rhel
It seems that it introduces the complete steps to setup passwordless ssh login, including permission issues

Reply:
Also this one:
https://phoenixnap.com/kb/setup-passwordless-ssh

Reply:
the script in autoyast runs inside guest, while 'ssh-copy-id' needs to be run in host. Unless you modify the unified guest installation base class to support this operation.

Reply:
Ignore ssh-copy-id, just manual part

Reply:
add more steps in autoyast about the permission of the ssh files.